easybuild/easyconfigs/s/split-seq/split-seq-20191107-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'split-seq'
+version = '20191107'
+local_commit = 'a711b56'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Alex-Rosenberg/split-seq-pipeline'
+description = """Analysis tools for split-seq."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/Alex-Rosenberg/split-seq-pipeline/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['%(name)s-%(version)s-add_pkl_barcodes.patch']
+checksums = [
+    '4432fd16ba40282fecada54287a990c81c4c829b7e7aee45e919b4c27219f319',  # split-seq-20191107.tar.gz
+    '0b85449eae71de94a3ba7b2a2533883618e18f2bfa1e6c6089a470165eb9f22c',  # split-seq-20191107-add_pkl_barcodes.patch
+]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('matplotlib', '3.0.0', versionsuffix),
+    ('STAR', '2.6.1c'),
+    ('SAMtools', '1.9'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%%(pyshortver)s/site-packages/%s' % name.replace('-', '_')],
+}
+
+modloadmsg = "Reference database located in /databases/bio/%(name)s-%(version)s\n"
```

Comment:
this is site-specific...

Reply:
Absolutely, I updated it to be site agnostic and commented it out as well